metashare/api/sf_run_flow.py

```diff
@@ -64,15 +64,26 @@ def refresh_access_token(*, config, org_name, scratch_org):
         org_config.config["access_token"] = info["access_token"]
         return org_config
     except HTTPError as err:
-        error_msg = "Are you certain that the org still exists?"
-
         if get_current_job():
             job_id = get_current_job().id
-            error_msg += f" If you need support, your job ID is {job_id}."
+            # This error is user-facing, and so for makemessages to
+            # pick it up correctly, we need it to be a single,
+            # unbroken, string literal (even though adjacent string
+            # literals should be parsed by the AST into a single
+            # string literal and picked up by makemessages, but
+            # that's a gripe for another day). We have relatively
+            # few errors that propagate directly from the backend
+            # like this, but when we do, this is the pattern we
+            # should use.
+            #
+            # This is also why we repeat the first sentence.
```

Comment:
The first sentence could be set up as a separate message, e.g. _("sentence 1") + _("sentence 2")

Reply:
Alas, you can't concatenate the results of from `django.utils.translation.gettext_lazy`.

Reply:
Well that's disappointing!

Reply:
I know, right? I wonder if the non-lazy version gets around that, though. Just occurred to me.